pkg/operator/operator.go

```diff
@@ -151,6 +152,19 @@ func New(config operatorconfig.Config, dnsManager dns.Manager, kubeConfig *rest.
 		return nil, fmt.Errorf("failed to create cacert controller: %v", err)
 	}
 
+	// Set up the certificate-publisher controller
+	certPublisherController, err := certpublishercontroller.New(operatorManager, kubeClient, config.Namespace)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create certificate-publisher controller: %v", err)
+	}
+	informer, err := operandCache.GetInformer(&corev1.Secret{})
+	if err != nil {
+		return nil, fmt.Errorf("failed to create informer for %T: %v", &corev1.Secret{}, err)
+	}
+	if err := certPublisherController.Watch(&source.Informer{Informer: informer}, &handler.EnqueueRequestForObject{}); err != nil {
```

Comment:
Convention with the other controllers is to internalize the `Watch` calls to the constructor (i.e. the controller should for the most part know what it wants to watch) — if you went with that factoring, the informer would be a parameter to `New`

Reply:
Fixed.  Thanks! I thought the signature of `New` was supposed to be consistent across controllers, but I guess there's no such requirement or convention.